common/character_interactions/00_blackmail_interactions.txt

```diff
@@ -12,7 +12,8 @@
 	is_shown = {
```

Comment:
Noticed this, prob we need to adapt this too.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/113774329-2c2c5e00-9738-11eb-81ea-18ff63b9c230.png)

</details>

Reply:
Prob it's this part
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/113774740-bbd20c80-9738-11eb-8b07-2c2df896e57b.png)

</details>

Reply:
Fixed in 9d5d3133251aaac30b1873526fc34dd9cfbbc8b6.